### PR TITLE
Announce artifact repository upgrade

### PR DESCRIPTION
**Pull request by MarkEWaite**
Another location to share the information.

See also https://github.com/jenkins-infra/.github/pull/29 

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [x] Link to relevant pull requests, esp. upstream and downstream changes
- [x] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ In a nutshell, Jenkins is the leading open-source automation server.
 It provides over 1,800 [plugins](https://plugins.jenkins.io/) for many use cases so that humans can spend their time doing things machines cannot.
 Learn more: [Jenkins Website](https://www.jenkins.io/), [Documentation](https://www.jenkins.io/doc/), [Plugin Index](https://plugins.jenkins.io/)
 
+## Scheduled downtime for Jenkins Artifactory and ci.jenkins.io
+
+The Jenkins artifact repository, [repo.jenkins-ci.org](https://repo.jenkins-ci.org/), will be upgraded beginning Sunday, December 18, 2022 at 9:00 AM UTC.
+Downtime is expectd to be 6 hours or less.
```
Comment (NotMyFault) [inline profile/README.md:10]:
```suggestion
Downtime is expected to be 6 hours or less.
```

**Review by NotMyFault — APPROVED** [review]
Otherwise, lgtm 👍🏻

**Review by lemeurherve — APPROVED** [review]
Same as the other PR, I would have put this at the bottom, fine as it is anyway.

**Comment by lemeurherve** [comment]
Preview from jenkins-infra:

image

**Comment by MarkEWaite** [comment]
> Same as the other PR, I would have put this at the bottom, fine as it is anyway.

I was trying to stay "above the fold" for readers.  I realize that  it disrupts the flow of the rest of the text, but wanted to increase the chances of the text being read.

**Comment by MarkEWaite** [comment]
If bottom of the page increases chance of merge, I'm happy to move it to the bottom of the page.  Page location is much less important to me than existence on the page.

File: profile/README.md [hunk 2]
```diff
@@ -4,6 +4,12 @@ In a nutshell, Jenkins is the leading open-source automation server.
 It provides over 1,800 [plugins](https://plugins.jenkins.io/) for many use cases so that humans can spend their time doing things machines cannot.
 Learn more: [Jenkins Website](https://www.jenkins.io/), [Documentation](https://www.jenkins.io/doc/), [Plugin Index](https://plugins.jenkins.io/)
 
+## Scheduled downtime for Jenkins Artifactory and ci.jenkins.io
+
+The Jenkins artifact repository, [repo.jenkins-ci.org](https://repo.jenkins-ci.org/), will be upgraded beginning Sunday, December 18, 2022 at 9:00 AM UTC.
+Downtime is expected to be 6 hours or less.
+More information is available on [status.jenkins.io](https://status.jenkins.io/issues/2022-12-09-artifactory-maintenance/) and in the [announcement blog post](https://www.jenkins.io/blog/2022/12/09/jfrog-maintenance/).
+
```
Comment (lemeurherve) [inline profile/README.md:12]:
```suggestion

---
```
Adding a line at the end of this announcement would help distinguish the different sections.
As it is I find it not clear.

**Review by lemeurherve — APPROVED** [review]
Suggesting a better separation, not blocking

File: profile/README.md (hunk 2 above)
Comment (lemeurherve) [reply profile/README.md:12]: Corresponding PR for jenkins-infra: https://github.com/jenkins-infra/.github/pull/30

**Pull request merged**